packages/react/src/components/Table/TableBody/TableBodyLoadMoreRow/TableBodyLoadMoreRow.test.jsx

```diff
@@ -13,12 +13,21 @@ const tableRowProps = {
   isLoadingMore: false,
 };
 
+const tableRowPropsLoading = {
```

Comment:
Since these are not "real constants" in the sense that the objects can be modified and since they are declared in the global scope there is a risk that one test could modify the test data and have it affect the other tests. So, even though this was already here before this PR I would suggest a small refactor to use functions that return data objects instead e.g.

```
const getTableRowPropsNotLoading = () => ({
  id: 'tableRow01',
  tableId: 'tableId',
  testId: 'table-test',
  totalColumns: 5,
  loadMoreText: 'Load more...',
  isLoadingMore: false,
});
```

You can also shallow copy the `tableRowPropsNotLoading` object and just change the attribute `isLoadingMore`:

`const getTableRowPropsLoading = () => ({...getTableRowPropsNotLoading(), isLoadingMore: true});`